start-site/src/main/resources/templates/spring-cloud-netflix-maintenance-mode.mustache

```diff
@@ -11,4 +11,4 @@ support for.
 
 Please see [this blog entry](https://spring.io/blog/2018/12/12/spring-cloud-greenwich-rc1-available-now#spring-cloud-netflix-projects-entering-maintenance-mode)
 for more information on maintenance mode and a list of suggested replacements for those
-libraries.
\ No newline at end of file
+libraries.
```

Comment:
This is an unrelated change to the purpose of this PR. Can you please revert that? Not adding a new line is on purpose as we prefer spaces to be handled consistently when all sections are rendered.

Reply:
Sure. I think I was getting a checkstyle or formatting validation issue here, but not anymore.